test/Serialization/setup_tests.jl

```diff
@@ -1,41 +1,47 @@
+using JSONSchema, JSON
```

Comment:
```suggestion
using JSONSchema, Oscar.JSON
```
this way you could get rid of the new test dependency. The `test/Project.toml` is intended to be used for test-only dependencies.